React/Views/RCTSliderManager.m

```diff
@@ -30,6 +30,13 @@ - (UIView *)view
 
 static void RCTSendSliderEvent(RCTSlider *sender, BOOL continuous)
 {
+  if (sender.step > 0 &&
+    sender.step <= (sender.maximumValue - sender.minimumValue)) {
+    int step = ((int)((sender.value + sender.step / 2) / sender.step) * sender.step);
```

Comment:
On second thought, that may not handle minimumValue/maximumValue correctly if they aren't a multiple of the step value. I'm not sure whether you'd ever have that situation, but in any case, I think the correct logic is:
    
    float value = MIN(sender.maximumValue, sender.minimumValue + round((sender.value - sender.minimumValue) / sender.step) * sender.step);